.github/workflows/docs.yaml

```diff
@@ -28,21 +32,40 @@ jobs:
           extra-packages: any::pkgdown, local::.
           needs: website
 
-      - name: Build pkgdown site
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          enable-cache: true
+          cache-dependency-glob: python/pyproject.toml
+          cache-suffix: docs
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version-file: python/pyproject.toml
+
+      - name: Install Python dependencies
+        working-directory: python
+        shell: bash
+        run: uv pip install .[docs]
+
+      - name: Build R docs
         run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)
         shell: Rscript {0}
 
+      - name: Build Python docs
+        run: sphinx-build python/docs/source docs/python
```

Comment:
Building these docs into the `docs/python` subdirectory makes them show up under the relative path `/python` in the deployed docs site.

Reply:
You _might_ be able to do client-side redirects with [http-equiv](https://www.w3.org/TR/WCAG20-TECHS/H76.html).